bigchaindb/events.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum
+from multiprocessing import Queue
+
+
+class EventTypes(Enum):
+    BLOCK_VALID = 1
+    BLOCK_INVALID = 2
+
+
+class Event:
+
+    def __init__(self, event_type, event_data):
+        self.type = event_type
+        self.data = event_data
+
+
+class EventHandler:
+
+    def __init__(self, events_queue):
+        self.events_queue = events_queue
+
+    def put_event(self, event, timeout=None):
+        # TODO: handle timeouts
```

Comment:
Personally i use TODO to indicate something i need to address in a PR. If there are TODOs in master its no problem, i will choose another random word to use.